docs/ecosystem/anyscale.md

```diff
@@ -0,0 +1,16 @@
+# Anyscale
+
+This page covers how to use the Anyscale ecosystem within LangChain.
+It is broken into two parts: installation and setup, and then references to specific Anyscale wrappers.
+
+## Installation and Setup
+- Get an Anyscale Service URL, route and API key and set them as environment variables (`ANYSCALE_SERVICE_URL`,`ANYSCALE_SERVICE_ROUTE`, `ANYSCALE_SERVICE_TOKEN`)
```

Comment:
Any good links we can provide here? e.g. to a Get Started page for Anyscale or something like that

Reply:
Sure. Just added the docs link and also reformatted anyscale.py with Black